src/main/java/de/slimecloud/slimeball/main/SlimeBot.java

```diff
@@ -319,6 +331,15 @@ public void onReady(@NotNull ReadyEvent event) {
 		new HolidayAlert(this);
 		new BirthdayAlert(this);
 		new BirthdayListener(this);
+
+		//botPackage
+		loadSubPackages(botPackage);
```

Comment:
Why doesn't JEvent load the packages itself?

Reply:
because you have more control over which packages should be loaded and when

Reply:
Then add a boolen parameter to the register method to indicate whether to automatically load all subpackages or not. It feels like this is something you want to do very commonly so it doesn't feel right to have ti do it manually here.

Reply:
We might even want to add ReflectClass#loadReflective to ReflectionUtils

Reply:
I will add a method in ReflectionUtils for this